### ISSUE TITLE
update Puzzle 3 answer

### ISSUE THREAD

**Issue by cschois1** [iamsammak/thehuntgame#174]
Puzzle 3's answer will be "ATGPSMJOR". It doesn't need to be all caps, so I guess an all-lower case or mixed case answer should be accepted.

**Comment by ihurrahi** [comment]
should puzzle 3 only have a text input rather than the current numeric keypad?

**Comment by iamsammak** [comment]
yeah we still need to refactor puzzle 3 and remove the keypad (sorry @pewpewpowpow)

**Comment by cschois1** [comment]
Yup, it should be only a text input and no keypad. Thanks!
